test/integration/worker/worker.js

```diff
@@ -16,15 +19,15 @@ describe('worker', () => {
     it('should return false and do nothing if no event is given', () => {
       let called = false;
       const reschedule = () => { called = true; };
-      runner({})(null, called).should.equal(false);
+      runner({})(null, reschedule).should.equal(false);
       called.should.equal(false);
     });
 
     it('should return false and do nothing if no jobs match the event', () => {
       let called = false;
       const reschedule = () => { called = true; };
       const event = { action: 'test.event' };
-      runner({}, { other: [ () => Promise.resolve(42) ] })(event, called).should.equal(false);
+      runner({}, { other: [ () => Promise.resolve(42) ] })(event, reschedule).should.equal(false);
       called.should.equal(false);
     });
```

Comment:
I changed these two tests, passing `reschedule` rather than `called` to the function returned by `runner()`. I think that function expects a callback.